src/main.py

```diff
@@ -58,8 +63,15 @@ def standby():
 
     Once launch is detected, this method will return.
     """
-    # TODO: Implement this state
-    pass
+
+    gpio = GPIOReader(False)
+    retrieve_data = gpio.retrieveData()
+
+    lsm = retrieve_data.__readLSM9DS1()
+
+    while vec_len(lsm.acceleration) < LAUNCH_ACCELERATION_THRESHOLD:
+        lsm = retrieve_data.__readLSM9DS1()
```

Comment:
The latest changes look good! The one thing that I just noticed now is that you are retrieving data from gpio, and then calling readLSM on that returned data. The LSM reading method is on the GPIOReader object itself.

So you can get rid of the retrieveData() call all together.

It's getting late and my brain isn't really at peak performance right now, but it might look something like this:

```suggestion
    gpio = GPIOReader(False)

    accel = gpio.__readLSM9DS1().acceleration
    while vec_len(accel) < LAUNCH_ACCELERATION_THRESHOLD:
        accel = gpio.__readLSM9DS1().acceleration
```

or

```suggestion
    gpio = GPIOReader(False)

    while True:
        accel = gpio__readLSM9DS1().acceleration
        if vec_len(accel) >= LAUNCH_ACCELERATION_THRESHOLD:
            break
```

I can't decide which is better right now.